mc crash when conflicting flags are set

## Expected behavior

`mc` should return an error message if invalid flags are set.

## Actual behavior

`mc` crashes and dumps a traceback:

```
$ mc admin trace -v -a --status-code 400 minio2
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xc0 pc=0x4a269c0]

goroutine 1 [running]:
github.com/minio/mc/cmd.matchTrace({{0xc000381170, 0x1, 0x1}, {0x5731098, 0x0, 0x0}, {0x5731098, 0x0, 0x0}, {0x5731098, ...}, ...}, ...)
	github.com/minio/mc/cmd/admin-trace.go:162 +0x2a0
github.com/minio/mc/cmd.mainAdminTrace(0xc00051f6c0)
	github.com/minio/mc/cmd/admin-trace.go:318 +0xbfb
github.com/minio/cli.HandleAction({0x4bd7180?, 0x4e0e538?}, 0x5?)
	github.com/minio/cli@v1.22.0/app.go:498 +0x5a
github.com/minio/cli.Command.Run({{0x4d7647c, 0x5}, {0x0, 0x0}, {0x0, 0x0, 0x0}, {0x4d9fea3, 0x20}, {0x0, ...}, ...}, ...)
	github.com/minio/cli@v1.22.0/command.go:225 +0x9f7
github.com/minio/cli.(*App).RunAsSubcommand(0xc00051f040, 0xc00051eb60)
	github.com/minio/cli@v1.22.0/app.go:387 +0x85b
github.com/minio/cli.Command.startApp({{0x4d75a95, 0x5}, {0x0, 0x0}, {0x0, 0x0, 0x0}, {0x4d8a328, 0x14}, {0x0, ...}, ...}, ...)
	github.com/minio/cli@v1.22.0/command.go:313 +0x689
github.com/minio/cli.Command.Run({{0x4d75a95, 0x5}, {0x0, 0x0}, {0x0, 0x0, 0x0}, {0x4d8a328, 0x14}, {0x0, ...}, ...}, ...)
	github.com/minio/cli@v1.22.0/command.go:111 +0x418
github.com/minio/cli.(*App).Run(0xc00051e820, {0xc000134000, 0x8, 0x8})
	github.com/minio/cli@v1.22.0/app.go:261 +0x847
github.com/minio/mc/cmd.Main({0xc000134000?, 0x8, 0x8})
	github.com/minio/mc/cmd/main.go:137 +0x365
main.main()
	github.com/minio/mc/main.go:27 +0x2e
```

## Steps to reproduce the behavior

Add both the `-a` flag and the `--status-code 400` flag

## mc --version

```
$ mc --version
mc version RELEASE.2022-08-05T08-01-28Z (commit-id=351d021b924b4d19f1eb716b9e2bd74644c402d8)
Runtime: go1.19 darwin/amd64
Copyright (c) 2015-0000 MinIO, Inc.
License GNU AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>
```

## System information
OSX 12.4, `mc` installed via `brew`

--- a/cmd/admin-trace.go
+++ b/cmd/admin-trace.go
@@ -159,7 +159,7 @@ func matchTrace(opts matchOpts, traceInfo madmin.ServiceTraceInfo) bool {
 	if len(opts.statusCodes) > 0 {
 		matched := false
 		for _, code := range opts.statusCodes {
-			if traceInfo.Trace.HTTP.RespInfo.StatusCode == code {
+			if traceInfo.Trace.HTTP != nil && traceInfo.Trace.HTTP.RespInfo.StatusCode == code {
 				matched = true
 				break
 			}
@@ -174,7 +174,7 @@ func matchTrace(opts matchOpts, traceInfo madmin.ServiceTraceInfo) bool {
 	if len(opts.methods) > 0 {
 		matched := false
 		for _, method := range opts.methods {
-			if traceInfo.Trace.HTTP.ReqInfo.Method == method {
+			if traceInfo.Trace.HTTP != nil && traceInfo.Trace.HTTP.ReqInfo.Method == method {
 				matched = true
 				break
 			}

